Atlantic 2024 CFP dates and text (#2096)
writethedocs/www: docs/conf/atlantic/2024/news/announcing-cfp.rst

.. post:: Feb 22, 2024
   :tags: {{shortcode}}-{{year}}, cfp

Announcing Call for Proposals
=============================

Today we are announcing our `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_.

Speaking at Write the Docs Atlantic is a fantastic way to share your ideas with our inspiring global community of documentarians.
If there is something you’d really like to see a talk on, submit a proposal on it, or refer someone else who you think would be good!

Sign up to the `Atlantic conference mailing list <https://www.writethedocs.org/newsletter/>`_ to be the first to hear details as we announce them.

Call for Proposals
------------------

Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!

**The conference is virtual and all talks will be pre-recorded, allowing you to present and attend from anywhere!**

> Our video producer will be available to help with recording if you want it.

Reviewer: ```suggestion
Our video producer will be available to help with recording, at your request.
```

Reviewer: Or something else. "if you want it" just felt out of tone with the rest of it
